requirements.txt

```diff
@@ -1,2 +1,6 @@
+--find-links https://download.pytorch.org/whl/cu113/torch_stable.html
```

Comment:
If this is a way of installing fastfold to the current environment, remind it in readme.md

Reply:
fixed